src/main/java/com/bankofspring/model/AccountTransactions.java

```diff
@@ -0,0 +1,41 @@
+package com.bankofspring.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Singleton class to record transactions
+ * @author Malcolm.Murray
+ *
+ */
+public class AccountTransactions {
+
+	private List<AccountTransaction> transactions;
+	
+	private static AccountTransactions instance;
```

Comment:
Ah, I thought you had a single `AccountTransaction`, that would have been silly.